ddtrace/contrib/integration_registry/_registry_schema.json

```diff
@@ -22,17 +22,25 @@
         "type": "object",
         "description": "Min, max, and list of tested versions for a specific dependency.",
         "properties": {
-          "min": {
-            "description": "Minimum explicit version found in test lockfiles.",
+          "tested_min": {
+            "description": "Minimum version found in test lockfiles.",
             "type": "string"
           },
-          "max": {
-            "description": "Maximum explicit version found in test lockfiles.",
+          "tested_max": {
+            "description": "Maximum version found in test lockfiles.",
+            "type": "string"
+          },
+          "supported_min": {
+            "description": "The minimum version supported by this integration. While we typically test specific versions (e.g., 5.3), support is implied for the entire major version (e.g., 5.0+). This field represents the lowest major version where compatibility was tested.",
+            "type": "string"
+          },
+          "supported_max": {
+            "description": "The upper boundary of versions supported by this integration. Value is 'latest' when the integration supports all future versions, or, when capped, specifies the highest tested compatible version.",
             "type": "string"
           }
         },
         "required": [
-          "min", "max"
+          "tested_min", "tested_max", "supported_min", "supported_max"
```

Comment:
Should we try to capture exact tested versions?

Since, maybe tested_min is 2.0 and tested_max is 4.0, but if we never test any 3.0 version, what is our confidence level? or would someone read and assume that we do test 3.0 with a min/max like that?

vs `tested_versions: [2.3.4, 4.5.1]` is a little more explicit, especially if we ever broadcast to customers.

Reply:
We definitely could do this and have all versions that we test, though I'm sure we probably miss some majors currently, eg we test major `2` and major `4` but no `3`.